packages/react-conformance/src/utils/getCallbackArguments.test.ts

```diff
@@ -295,7 +295,7 @@ describe('getCallbackArguments', () => {
       expect(() =>
         getCallbackArguments(program, 'Accordion.types.ts', 'AccordionProps', 'onToggle'),
       ).toThrowErrorMatchingInlineSnapshot(
-        `"We received a type \\"Pick<TypeA, \\"open\\">\\" that is too complex to resolve. Please simply it, for example remove usage of \\"Pick\\"."`,
+        `"We received a type \\"TypeB\\" that is too complex to resolve. Please simply it, for example remove usage of \\"Pick\\"."`,
```

Comment:
@layershifter Do you think this is okay? Pretty sure it's a result of reductions in type expansion.

Reply:
It still reports the problem, so let's leave it as it is.